frontend/.eslintrc.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
 		es6: true,
 	},
```

Comment:
Reproduction steps:
In general this happens for the 1st edge of the model.

1. Load `simple.fold`
2. Select the edge in the middle of the model
![image](https://user-images.githubusercontent.com/23579650/93597312-213bf700-f9bb-11ea-9124-75817a02e00c.png)

3. Change anything. Assignment or target angle.

```
Uncaught TypeError: _useStore2$ is undefined
    GuideCreator index.js:13
    React 11
    unstable_runWithPriority scheduler.development.js:653
    React 10
    unstable_runWithPriority scheduler.development.js:653
    React 23
    js index.js:52
    Webpack 5
index.js:13:39
The above error occurred in the <GuideCreator> component:
    in GuideCreator (created by AppWithStore)
    in Route (created by AppWithStore)
    in StoreProvider (created by AppWithStore)
    in Router (created by HashRouter)
    in HashRouter (created by AppWithStore)
    in AppWithStore

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://fb.me/react-error-boundaries to learn more about error boundaries. react-dom.development.js:19527
```